sdk/typescript/runtime/main.go

```diff
@@ -31,6 +31,17 @@ const (
 	Node SupportedTSRuntime = "node"
 )
 
+type SupportedPackageManager string
+
+const (
+	Yarn       SupportedPackageManager = "yarn"
+	Pnpm       SupportedPackageManager = "pnpm"
+	Npm        SupportedPackageManager = "npm"
+	BunManager SupportedPackageManager = "bun"
+)
+
+const YarnDefaultVersion = "1.22.22"
```

Comment:
Why default to v1 when there's other more [modern versions](https://yarnpkg.com/migration/overview)?

Reply:
Just noticed the comment in the description about Yarn v4. What about v3, have you tried it?

Reply:
v3 has the same issue yep